### ISSUE TITLE
[Bug]: Signing in the wrong user when authenticating through a generic OAuth provider

### ISSUE THREAD

**Issue by nangelov-eti** [lowcoder-org/lowcoder#1396]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

User B is signed in as User A, causing incorrect user authentication.

### Expected Behavior

User B should be signed in as themselves after authenticating via the OAuth provider, not as User A.

### Steps to reproduce

1. Set up a generic OAuth provider.
2. Sign in using the OAuth provider with User A.
3. Open an incognito window and attempt to sign in with User B using the same OAuth provider.
4. Observe that User B is signed in as User A instead of their own account.

### Environment

Lowcoder version 2.4.10 is affected, but it's likely that the issue also impacts the latest version.
mode: ENTERPRISE

### Additional Information

We use Keycloak as the OAuth provider to authenticate across multiple Keycloak realms. However, since Lowcoder doesn’t support multiple Keycloak providers, we had to configure separate generic OAuth providers, each representing a different Keycloak realm.

**Comment by FalkWolsky** [comment]
This is never reported before and we cannot recreate this behaviour. If we would sign in the wrong users - the whole community of users would scream fast...  We will close the bug issue as for now as we are pretty sure this is not an existing bug. However, please use our discord to open your topic in a support channel - so we can have a look into your issue and understand the root cause. https://discord.com/invite/An6PgWpegg

**Comment by nangelov-eti** [comment]
You are right! It is a misconfiguration on our end. We had missing mappers (`uid`, `username`, `email`, `picture`) in Keycloak on that environment. Once the mappers were set, everything worked smoothly, but I feel like even if there is a misconfiguration, Lowcoder should have returned an error instead of signing me in as another user. Thank you for your hard work!
